### ISSUE TITLE
апдейт

### ISSUE THREAD

**Issue by Van4er85** [ivan02022000/HCWebKeyGen#1]
будет апдейт под новый игры ???

**Comment by ivan02022000** [comment]
Да, было обновление, добавлены 2 новые игры. Также изменены настройки ввода- ввод будет происходить дольше (около получаса) для имитации ручного ввода.
